[2023-12-20T11:50:41.418+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2022-04-27T00:00:00+00:00 [queued]>
[2023-12-20T11:50:41.514+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2022-04-27T00:00:00+00:00 [queued]>
[2023-12-20T11:50:41.515+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2023-12-20T11:50:41.605+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_id> on 2022-04-27 00:00:00+00:00
[2023-12-20T11:50:41.620+0000] {standard_task_runner.py:60} INFO - Started process 6284 to run task
[2023-12-20T11:50:41.648+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_id', 'scheduled__2022-04-27T00:00:00+00:00', '--job-id', '1527', '--raw', '--subdir', 'DAGS_FOLDER/first_dat.py', '--cfg-path', '/tmp/tmpgv5bcrwe']
[2023-12-20T11:50:41.652+0000] {standard_task_runner.py:88} INFO - Job 1527: Subtask second_id
[2023-12-20T11:50:41.839+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.second_id scheduled__2022-04-27T00:00:00+00:00 [running]> on host 29b3723064ac
[2023-12-20T11:50:42.215+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abrham' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_id' AIRFLOW_CTX_EXECUTION_DATE='2022-04-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-27T00:00:00+00:00'
[2023-12-20T11:50:42.217+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-20T11:50:42.217+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be runing after task 3']
[2023-12-20T11:50:42.245+0000] {subprocess.py:86} INFO - Output:
[2023-12-20T11:50:42.250+0000] {subprocess.py:93} INFO - hey, I am task2 and will be runing after task 3
[2023-12-20T11:50:42.250+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-20T11:50:42.329+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_id, execution_date=20220427T000000, start_date=20231220T115041, end_date=20231220T115042
[2023-12-20T11:50:42.400+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-20T11:50:42.445+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
